tools/MarkdownConverter/Grammar/Production.cs

```diff
@@ -7,6 +7,11 @@ internal class Production
         /// </summary>
         public EbnfNode Ebnf { get; set; }
 
+        /// <summary>
+        /// Whether or not this production is a fragment
+        /// </summary>
+        public bool Fragment { get; set; }
```

Comment:
Not necessary, but it looks like this could be an init only property once we move to C# 9. I only see it set in the property initializer.

```suggestion
        public bool Fragment { get; init; }
```

Reply:
Yes, I suspect that may well be fine. I was mostly following the rest of the properties. I'd prefer to leave it as it is for now, and then revisit holistically.